plugins/module_utils/redfish_utils.py

```diff
@@ -3051,3 +3052,62 @@ def get_manager_inventory(self, manager_uri):
 
     def get_multi_manager_inventory(self):
         return self.aggregate_managers(self.get_manager_inventory)
+
+    def set_sessionservice(self, sessionservice_config):
+        result = {}
+        response = self.get_request(self.root_uri + self.session_service_uri)
+        if response['ret'] is False:
+            return response
+        current_sessionservice_config = response['data']
+        payload = {}
+        for property in sessionservice_config.keys():
+            value = sessionservice_config[property]
+            if property not in current_sessionservice_config:
+                return {'ret': False, 'msg': "Property %s in sessionservice_config is invalid" % property}
+            if isinstance(value, dict):
+                if isinstance(current_sessionservice_config[property], dict):
+                    payload[property] = value
+                elif isinstance(current_sessionservice_config[property], list):
+                    payload[property] = list()
+                    payload[property].append(value)
+                else:
+                    return {'ret': False, 'msg': "Value of property %s in sessionservice_config is invalid" % property}
+            else:
+                payload[property] = value
+
+        need_change = False
+        for property in payload.keys():
+            set_value = payload[property]
```

Comment:
```suggestion
        for property, set_value in payload.items():
```

Reply:
Hi @russoz! Thanks for your feedback! So, I took this code snippet from the set_hostinterface_attributes() which seems to work as expected: https://github.com/ansible-collections/community.general/blob/49268d13ff4260bf23f69e203c6c41c8674f8adc/plugins/module_utils/redfish_utils.py#L2908-L2915
Let me know if you'd like any specific changes for this, and I'd be happy to implement them.

Reply:
I think you responded on a different comment. I will respond to this over there.